devfiles/nodejs-react/devfile.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: nodejs-react
+projects:
+  -
+    name: react-redux-realworld-example-app
+    source:
+      type: git
+      location: "https://github.com/gothinkster/react-redux-realworld-example-app"
+components:
+  -
+    type: chePlugin
+    id: che-incubator/typescript/latest
+    memoryLimit: 512Mi
+  -
+    type: dockerimage
+    alias: nodejs
+    # system limit for number of file watchers reached with
+    # image: registry.access.redhat.com/ubi8/nodejs-10
```

Comment:
Is this issue logged/tracked somewhere?

Reply:
@rhopp 
No. Should issue be created in eclipse/che repo?